addon/components/learning-materials-sort-manager.hbs

```diff
@@ -42,10 +42,8 @@
               </span>
               <span class="details">
                 {{capitalize item.learningMaterial.type}},
-                {{t
-                  "general.ownedBy"
-                  owner=item.learningMaterial.owningUser.fullName
```

Comment:
Did you try `owner=(item.learningMaterial.owningUser.fullName)`? I hate this change, but can accept it if necessary.

Reply:
agreed, i'm not happy about this either. i tried a couple of things to keep the interpolation, but can't seem to get it to work in the template alone. pls lmk if you have a better idea.